apps/survey/src/components/prompts/standard/SubmitPrompt.vue

```diff
@@ -1,7 +1,35 @@
 <template>
   <card-layout v-bind="{ food, meal, prompt, section, isValid }" @action="action">
+    <review-meal-list
```

Comment:
use <component :is="myComponentNameValue"> to selelct component based on conditions, lots of repetition.

Do we also need component for each layout or behaviour logic? I think we said that on mobile it practically the same? If it cannot fit to less components, we should extract the common logic at least.

Reply:
I have kept only two final review list components—one for the desktop and the other for the mobile versions. I use v-if and some flags to switch between the checkbox and the scroll bar options.